libr/bin/p/bin_mbn.c

```diff
@@ -77,14 +77,14 @@ static bool load_bytes(RBinFile *bf, void **bin_obj, const ut8 *buf, ut64 sz, ut
 
 static bool load(RBinFile *bf) {
 	if (bf && bf->buf) {
-		const ut8 *bytes = r_buf_buffer (bf->buf);
-		ut64 sz = r_buf_size (bf->buf);
+		ut64 sz;
+		const ut8 *bytes = r_buf_buffer (bf->buf, &sz);
```

Comment:
i think r_buf_buffer is a really bad name

Reply:
`r_buf_data`?